dll/win32/shell32/CUserNotification.cpp

```diff
@@ -19,6 +19,8 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
  */
 
+#undef _WIN32_WINNT
+#define _WIN32_WINNT _WIN32_WINNT_VISTA // NOTIFYICONDATAW_V3_SIZE needs Vista+
```

Comment:
Re-adding such a hack kind of defeat the purpose of this PR.
Vista+ code must be adapted so it can be built as XP-S2003 too...

Reply:
PSDK doesn't supports `NOTIFYICONDATAW_V3_SIZE` in pre-Vista. No way.

Reply:
Is there any reason to not structurally use V2 everywhere?

Reply:
V# is fixed by `compat.h`.

Reply:
> Is there any reason to not structurally use V2 everywhere? And 2nd question: if you say that v3 is not Vista, but XP/2k3 still.... why can't it be added to the PSDK then? I don't understand.

You are confusing when and why these sizes are defined: see #5730.

Reply:
Current Windows SDK has this define and it's not wrapped in any #ifdefs. And there is no argument we couldn't add to our psdk headers. We generally target latest SDK, not XP SDK.